t/03-dns-client/01-phases.t

```diff
@@ -59,8 +59,7 @@ qq {
 GET /t
 --- response_body
 answers: nil
-err: dns client error: 101 empty record received
---- no_error_log
+err: nil
+--- error_log
```

Comment:
@chobits why did this test result change?

Reply:
This test case was introduced by this [PR](https://github.com/Kong/lua-resty-dns-client/pull/130/files) , and its scenario will not be used in kong project because this library has been imported into kong, we do not run dns.client query in init_worker phase.

So I eliminate asynchronous timer in syncQuery() in this PR, after this, the test case result will change. Now if you call dns query API in init_worker phase, it will be disabled directly.